knx/dpt/types_8..go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2017 Ole Krüger.
+// Licensed under the MIT license which can be found in the LICENSE file.
+
+package dpt
+
+import (
+	"fmt"
+)
+
+// DPT_8001 represents DPT 8.001 / Counter.
+type DPT_8001 int16
+
+func (d DPT_8001) Pack() []byte {
+	return packV16(int16(d))
+}
+
+func (d *DPT_8001) Unpack(data []byte) error {
+	return unpackV16(data, (*int16)(d))
+}
+
+func (d DPT_8001) Unit() string {
+	return ""
+}
+
+func (d DPT_8001) String() string {
+	return fmt.Sprintf("%d °", int16(d))
+}
+
+// DPT_8002 represents DPT 8.002 / delta time ms.
+type DPT_8002 int16
+
+func (d DPT_8002) Pack() []byte {
+	return packV16(int16(d))
+}
+
+func (d *DPT_8002) Unpack(data []byte) error {
+	return unpackV16(data, (*int16)(d))
+}
+
+func (d DPT_8002) Unit() string {
+	return "ms"
+}
+
+func (d DPT_8002) String() string {
+	return fmt.Sprintf("%d ms", int16(d))
+}
+
+// DPT_8003 represents DPT 8.003 / delta time centiseconds
+type DPT_8003 int16
+
+func (d DPT_8003) Pack() []byte {
+	return packV16(int16(d))
+}
+
+func (d *DPT_8003) Unpack(data []byte) error {
+	return unpackV16(data, (*int16)(d))
+}
+
+func (d DPT_8003) Unit() string {
+	return "cs"
+}
+
+func (d DPT_8003) String() string {
+	return fmt.Sprintf("%d cs", int16(d))
+}
+
+// DPT_8004 represents DPT 8.004 / delta time deciseconds
```

Comment:
I'm wondering if this is actually correct. Spec seem to suggest that this value should be divided by 10, and DPT_8005 by 100. But it's the same with 7.003 and 7.004 and this library does not seem to treat those in any special way so not sure.

Reply:
Both of them are actually in seconds. But by calling the units centiseconds and deciseconds we combat the need for dividing them by 100 and 10 respectively. That's why I used those units.